apps/dashboard/src/routes/courses/[id]/lessons/[...lessonParams]/+page.svelte

```diff
@@ -373,6 +423,18 @@
     <div
       class="flex items-center gap-2 w-fit rounded-full shadow-xl bg-gray-100 dark:bg-neutral-700 px-5 py-1"
     >
+      <button
+        disabled={disabled.prev}
+        class={`px-2 my-2 pr-4 border-t-0 border-b-0 border-l-0 border border-gray-300 flex items-center ${
+          disabled.prev && 'cursor-not-allowed'
+        }`}
+        on:click={goToPrevLesson}
+      >
+        <ChevronLeft size={24} />
+        {#if !$isMobile}
+          <span> Prev </span>
+        {/if}
```

Comment:
You only use js for things that can't be done in css.

just use css to hide this element on mobile